src/components/action-popover/action-popover.stories.mdx

```diff
@@ -653,6 +654,34 @@ used actions, in this example "manage devices".
   </Story>
 </Preview>
 
+## in overflow hidden container
```

Comment:
We've got some odd `z-index` here 
![image](https://user-images.githubusercontent.com/2328042/102996975-f7dfb980-451b-11eb-8c64-5331b0b6c91a.png)

![image](https://user-images.githubusercontent.com/2328042/102997049-147bf180-451c-11eb-85b4-64d4fc76c401.png)